metaflow/plugins/aws/eks/kubernetes_client.py

```diff
@@ -239,7 +239,7 @@ def create(self):
                                 image=self._kwargs["image"],
                                 name=self._kwargs["name"],
                                 resources=client.V1ResourceRequirements(
-                                    requests={
+                                    limits={
                                         "cpu": str(self._kwargs["cpu"]),
                                         "memory": "%sM" % str(self._kwargs["memory"]),
                                         "ephemeral-storage": "%sM"
```

Comment:
should we be interpolating something here?

Reply:
I moved from `requests` to `limits` to bring it in line with the Argo implementation. Did I understand your comment correctly?

Reply:
We're not interpolating a value for the ephemeral storage currently, but it looks like we intend to?

Reply:
oh nvm, I was thrown by the formatting/git cutting off the diff.  Whoops.